src/ER_V/acl/acl.c

```diff
@@ -27,32 +27,67 @@ int ACL_Command_init(ACL_Command *cmd)
 
 int ACL_init()
 {
+    memset(&resources, 0, sizeof(resources));
     for(uint16_t iter = 0; iter < ACL_CMD_COUNT; iter++)
     {
         ACL_Command_init(&resources.cmd_pool[iter]);
     }
 
+    resources.manaul_mode = 0;  // Default is not manual mode
+    return 0;
+}
+
+int ACL_enqueue_manual_mode_toggle_cmd(S_List *cmd_queue)
+{
+    S_List_Node* node = DATA_S_List_pop(&resources.free_queue);
+    if (!node) STD_FAIL;
+
+    ACL_Command* cmd = DATA_LIST_GET_OBJ(node, ACL_Command, node);
+    cmd->type = ACL_CMD_MANUAL;
+    cmd->len = sprintf(&cmd->payload[0], ACL_TOGGLE_MANUAL_FMT);
+
+    DATA_S_List_append(cmd_queue, &cmd->node);
     return 0;
 }
 
+char ACL_get_polar_pan_continuous_vector(API_Data_Polar_Pan_Start* payload)
+{
+    ACL_Manual_Axis vector;
+
+    if (payload->delta_azimuth)
```

Comment:
Small nitpick but I think it would be cleaner to use curly braces for this whole if/else block